priv/signs.json

```diff
@@ -9179,7 +9179,21 @@
             "direction_id": 0
           }
         ]
-      },
+      }
+    ],
+    "max_minutes": 60
+  },
+  {
+    "id": "bus.Lechmere_outer",
+    "pa_ess_loc": "SLEC",
+    "read_loop_interval": 360,
+    "read_loop_offset": 0,
+    "text_zone": "c",
+    "audio_zones": [
+      "m"
```

Comment:
This configuration will perform two separate readouts on the same zone, offset by a couple minutes. I thought we were going to handle this similar to Nubian, where the visual messages are split but there's a single combined audio readout via the `extra_audio_configs` mechanism, so people don't get confused if they don't hear their bus.

Reply:
Ah right, I forgot we added that feature to the bus signs. Just adjusted to the config